test/unit/Exception/ModuleExceptionInterfaceTest.php

```diff
@@ -12,34 +14,20 @@
  */
 class ModuleExceptionInterfaceTest extends TestCase
 {
-    /**
-     * The name of the test subject.
-     *
-     * @since [*next-version*]
-     */
-    const TEST_SUBJECT_CLASSNAME = 'Dhii\Modular\Module\Exception\ModuleExceptionInterface';
+    use GetImplementingMockBuilderCapableTrait;
 
     /**
      * Creates a new instance of the test subject.
      *
      * @since [*next-version*]
      *
-     * @return TestSubject
+     * @return TestSubject|MockObject
      */
     public function createInstance()
     {
-        $mock = $this->mock(static::TEST_SUBJECT_CLASSNAME)
-            ->getMessage()
-            ->getCode()
-            ->getPrevious()
-            ->getFile()
-            ->getLine()
-            ->getTrace()
-            ->getTraceAsString()
-            ->__toString()
-
-            ->getModule()
-            ->new();
+        $mock = $this->getImplementingMockBuilder('Exception', [TestSubject::class])
```

Comment:
The use of the trait and this method is causing builds to fail.

@XedinUnknown This test does not require the test subject interface to be implemented by a mock that extends `Exception`. Any reason why this is being used?

Reply:
> This test does not require the test subject interface to be implemented by a mock that extends `Exception`. Any reason why this is being used?

It doesn't, but it should, because the subject must be throwable. Fixing...

Reply:
Why should it be throwable?

Reply:
Because it's something you... throw?

Reply:
This test is testing the interface, and nothing about the test nor the interface has anything to do with throwing.

Reply:
> the subject does not need to be throwable

Of course it needs to be throwable. You cannot `catch` something that is not throwable.

Reply:
Fixed.

Reply:
But you're not catching it. It's a bloody interface. And your test just checks if its module-aware.